### PR TITLE
Origin/phonebook

### PR DESCRIPTION
**Pull request by maximeroucher**
### Description

Please explain the changes you made here.

### Checklist

- [ ] Created tests which fail without the change (if possible)
- [ ] All tests passing
- [ ] Extended the documentation, if necessary

### CODE REVIEW

File: app/endpoints/phonebook.py [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+    status_code=200,
+    tags=[Tags.phonebook],
+)
+async def create_member(
+    association_id: str,
+    mandate_year: int,
+    role_id: str,
+    user_id: str,
+    db: AsyncSession = Depends(get_db),
+    requesting_user=Depends(is_user_a_member_of(GroupType.CAA)),
```
Comment (armanddidierjean) [inline app/endpoints/phonebook.py:175]: Shouldn't we pass the `Member` content as a Body instead of Query strings ?

File: requirements.txt
Comment (armanddidierjean) [inline requirements.txt:1]: Remove unneeded changes

**Review by armanddidierjean — CHANGES_REQUESTED** [review]

**Comment by armanddidierjean** [comment]
This PR needs to be rebased on main branch

**Comment by guillaumetavernier** [comment]
Updated PR on #342

**Pull request closed without merging**